docs/core/porting/versioning-sdk-msbuild-vs.md

```diff
@@ -7,43 +7,39 @@ ms.date: 06/10/2021
 ---
 # Overview of .NET, MSBuild, and Visual Studio versioning
 
-Understanding the versioning of the SDK and how it relates to Visual Studio and MSBuild can frequently be confusing. MSBuild versions with VS but is also included in the SDK. The SDK has a minimum version of MSBuild that it works with and it won't load in a VS that is older than it supports.
+Understanding the versioning of the SDK and how it relates to Visual Studio and MSBuild can frequently be confusing. MSBuild versions with VS but is also included in the SDK. The SDK has a minimum version of MSBuild/VS that it works with and it won't load in a VS that is older than that minimum version.
 
 ## Versioning
 
-The first part of the SDK matches the .NET version that it includes, runs on, and targets by default.  The feature band starts at 1 and increases for each quarterly VS minor release.  The patch version increments with each month's servicing updates.
+The first part of the SDK matches the .NET version that it includes, runs on, and targets by default.  The feature band starts at 1 and increases for each quarterly VS minor release.  The patch version increments with each month's servicing updates. 
 
-For example, 5.0.203 ships with .NET 5.0, is the second minor VS release since 5.0.100 first came out, and is the third patch since 5.0.200 released.
+For example, 5.0.203 ships with .NET 5.0, is the 2nd minor VS release since 5.0.100 first came out, and is the 3rd patch since 5.0.200 released.
```

Comment:
```suggestion
For example, version 5.0.203 ships with .NET 5.0, is the second minor Visual Studio release since 5.0.100 first came out, and is the third patch since 5.0.200 released.
```

Reply:
Numbers less than 10 should be spelled out, according to the style guide.